search_objects/src/search_objects.py

```diff
@@ -0,0 +1,25 @@
+import json
+import boto3
+from boto3.dynamodb.conditions import Attr, Or
+
+dynamodb = boto3.resource('dynamodb')
+table = dynamodb.Table('kutter-table')
+
+def search_objects_data(search, limit):
+    filter_expression = Or( Attr('creationDate').contains(search), Attr('originalFileName').contains(search))
+    response = table.scan(Limit = limit,FilterExpression = filter_expression) 
+    return { 
+         'items' :response['Items']
+    }
+
+def handler(event, context):
+    search = event ['queryStringParameters']['search']
+    limit = int(10)
+    items = search_objects_data(search,limit)
```

Comment:
you can also add pagination instead of fixt limit

Reply:
will be tracked in issue https://github.com/Chufle/kutter/issues/68